modules/dedicated-enable-private-cluster.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * cloud_infrastructure_access/dedicated-private-cluster.adoc
+
+[id="dedicated-enable-private-cluster-new"]
+= Enabling Private Cluster on a new cluster
+
+.Prerequisites
+
+* AWS VPC Peering, VPN, DirectConnect, or TransitGateway has been previously configured to allow private access.
+
+.Procedure
+
+. In the OpenShift Cluster Manager, click *Create cluster* and select *{product-title}*.
+. Configure your Cluster details, then select *Advanced* in the Networking section.
+. Determine your CIDR requirements for your network.
++
+[IMPORTANT]
+====
+CIDR configurations cannot be changed later. Confirm your selections with your network administrator before proceeding.
+====
+. Under Cluster Privacy, select *Private*.
+
+[id="dedicated-enable-private-cluster-existing"]
```

Comment:
Split this section into a separate module - you can have only one procedure section per procedure module, and this is a separate process.